### PR TITLE
user-review-history

### PR DESCRIPTION
**Pull request by hungsondo**
## Related Tickets:
- [#40705](https://edu-redmine.sun-asterisk.vn/issues/40705)
- [#40707](https://edu-redmine.sun-asterisk.vn/issues/40707)

## WHAT this PR does?
- Add template user profile
- In user profile has user's info + review history
- Auth user can see other user profile + review history

## Notes Impacted Areas:
- N/A.

## Attachments:
mp
up

### CODE REVIEW

**Review by longnv-99 — APPROVED** [review]

**Review by truongcr7 — APPROVED** [review]

**Comment by hungsondo** [comment]
ready

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -106,4 +108,28 @@ public function destroy($id)
             return redirect()->route('users.index')->with('error', __('messages.delete-user-failed'));
         }
     }
+
+    public function myProfile()
+    {
+        $reviewHistory = Review::where('user_id', '=', getAuthUserId())
+                            ->where('display', '=', config('app.display'))
+                            ->orderBy('updated_at', 'DESC')
+                            ->get();
```
Comment (manhtd-1083) [inline app/Http/Controllers/UserController.php:117]: indent 4 space

File: app/Http/Controllers/UserController.php [hunk 2]
```diff
@@ -106,4 +108,28 @@ public function destroy($id)
...
+
+        return view('profile', compact(['reviewHistory','user']));
+    }
+
+    public function getUserProfile($id)
+    {
+        $reviewHistory = Review::where('user_id', '=', $id)
+                            ->where('display', '=', config('app.display'))
+                            ->orderBy('updated_at', 'DESC')
+                            ->get();
```
Comment (manhtd-1083) [inline app/Http/Controllers/UserController.php:129]: tương tự

**Comment by hungsondo** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**